app/models/vm/operations/power.rb

```diff
@@ -7,10 +7,27 @@ module Vm::Operations::Power
     api_relay_method :suspend
 
     supports :suspend do
-      msg = unsupported_reason(:control) unless supports_control?
-      msg ||= _('The VM is not powered on') unless vm_powered_on?
+      msg = supports_control_powered_on
```

Comment:
I think the support blocks should be dead simple:

```ruby
supports :pause do
  unsupported_reason_add(:pause, unsupported_reason(:control)) unless supports_control?
  unsupported_reason_add(:pause, _('The VM is not powered on')) unless vm_powered_on?
end
```

This way you can easily grasp the preconditions. 
I see that we are duplicating those 2 lines, but thats a good tradeoff imho.